modules/ibi-generate-seed-image.adoc

```diff
@@ -0,0 +1,176 @@
+// Module included in the following assemblies:
+// * edge_computing/ztp-image-based-upgrade.adoc
+// * edge_computing/factory-image-based-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-image-based-upgrade-seed-generation_{context}"]
+= Generating a seed image with the {lcao}
+
+Use the {lcao} to generate the seed image with the `SeedGenerator` CR. The Operator checks for required system configurations, performs any necessary system cleanup before generating the seed image, and launches the image generation. The seed image generation includes the following tasks:
+
+* Stopping cluster operators
+* Preparing the seed image configuration
+* Generating and pushing the seed image to the image repository specified in the `SeedGenerator` CR
+* Restoring cluster operators
+* Expiring seed cluster certificates
+* Generating new certificates for the seed cluster
+* Restoring and updates the `SeedGenerator` CR on the seed cluster
+
+[NOTE]
+====
+The generated seed image does not include any site-specific data.
+
+To avoid including any {rh-rhacm-first} resources in your seed image, you need to disable all optional {rh-rhacm} add-ons for seed image generation.
+====
+
+.Prerequisites
+
+* You have deployed a managed a {sno} cluster with a DU profile from a hub cluster running {rh-rhacm}.
+* Install the {lcao} in the hub cluster.
+* Install the OADP Operator on the seed cluster.
+* Log in as a user with `cluster-admin` privileges.
+* Review the required versions of software components in the section _Software prerequisites for the image-based installation_.
+* The seed cluster requires an extra partition for precaching release container images.
```

Comment:
@eranco74 - we don't have any instructions on how to create a partition at the moment. Do we need to copy content from here: https://74025--ocpdocs-pr.netlify.app/openshift-enterprise/latest/edge_computing/image_based_upgrade/preparing_for_image_based_upgrade/cnf-image-based-upgrade-shared-container-image